executable.go

```diff
@@ -173,18 +178,17 @@ func _Execute( // nolint
 
 		node._InUse()
 
-		logCtx.Trace().Str("requestId", logID).Str("nodeAccountID", node.accountID.String()).Str("nodeIPAddress", node.address._String()).
-			Str("Request Proto:", hex.EncodeToString(marshaledRequest)).Msg("executing")
+		client.logger.Trace("executing", "requestId", logID, "nodeAccountID", node.accountID.String(), "nodeIPAddress", node.address._String(), "Request Proto", hex.EncodeToString(marshaledRequest))
 
 		if !node._IsHealthy() {
-			logCtx.Trace().Str("requestId", logID).Str("delay", node._Wait().String()).Msg("node is unhealthy, waiting before continuing")
-			_DelayForAttempt(logID, backOff.NextBackOff(), attempt)
+			client.logger.Trace("node is unhealthy, waiting before continuing", "requestId", logID, "delay", node._Wait().String())
```

Comment:
I think everywhere in the execute function we should use the `txLogger` defined above

Reply:
fair point